build_all.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eux
+
+for pb in $(find packagingbuild -name Dockerfile); do
+	path=$(dirname $pb)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/packagingbuild:$flavor .) || exit -1
+done
+
+for pt in $(find packagingtest -name Dockerfile); do
+	path=$(dirname $pt)
+	flavor=$(basename $path)
```

Comment:
I am not sure this is right. The logic to work out flavour looks identical between vyos and test - but they are different directory levels?

Reply:
I'm 99% sure you mean to write `build` instead of `vyos`.

And I merge in the cleanup from #101, which removed the `systemd` directories and moved their contents up one level.

So the Dockerfiles for both packagingbuild and packagingtest are now at the same depths from the repository root.

You can browse the repo as of this commit:
https://github.com/StackStorm/st2packaging-dockerfiles/tree/5d4e656068016e00e15aca50038968ea43103e4d/packagingbuild
https://github.com/StackStorm/st2packaging-dockerfiles/tree/5d4e656068016e00e15aca50038968ea43103e4d/packagingtest